[bug] Setup kind cluster step fails in the tilt CI run

**Describe the bug**

Currently the **Setup Kind Cluster** step fails in the Tilt CI run with the following error
```
Run ctlptl create cluster kind --registry=ctlptl-registry
Not connected to Docker Engine. Host: "unix:///var/run/docker.sock". Error: Error response from daemon: client version 1.41 is too old. Minimum supported API version is 1.44, please upgrade your client to a newer version
Error: Process completed with exit code 1.
```

Reference: https://github.com/guacsec/guac/actions/runs/22076728221/job/63793327921

--- a/.github/workflows/ci-integration.yaml
+++ b/.github/workflows/ci-integration.yaml
@@ -140,7 +140,7 @@ jobs:
               armv7l)  ARCH=arm;;
               *)       ARCH=$(uname -m);;
           esac
-          VERSION=0.32.0
+          VERSION=0.36.3
           curl -fsSL https://github.com/tilt-dev/tilt/releases/download/v$VERSION/tilt.$VERSION.linux.$ARCH.tar.gz | tar -xzvC /usr/local/bin tilt
 
           # helm
@@ -154,14 +154,14 @@ jobs:
           curl -fsSL https://get.helm.sh/helm-v$VERSION-linux-$ARCH.tar.gz | tar --strip-components=1 -xzvC /usr/local/bin linux-$ARCH/helm
 
           # ctlptl - https://github.com/tilt-dev/ctlptl/blob/main/INSTALL.md
-          CTLPTL_VERSION="0.8.19"
+          CTLPTL_VERSION="0.9.0"
           curl -fsSL https://github.com/tilt-dev/ctlptl/releases/download/v$CTLPTL_VERSION/ctlptl.$CTLPTL_VERSION.linux.x86_64.tar.gz | sudo tar -xzv -C /usr/local/bin ctlptl
 
           # kind - https://kind.sigs.k8s.io/docs/user/quick-start/#installing-from-release-binaries
           # For AMD64 / x86_64
-          [ $(uname -m) = x86_64 ] && curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.19.0/kind-linux-amd64
+          [ $(uname -m) = x86_64 ] && curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.31.0/kind-linux-amd64
           # For ARM64
-          [ $(uname -m) = aarch64 ] && curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.19.0/kind-linux-arm64
+          [ $(uname -m) = aarch64 ] && curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.31.3/kind-linux-arm64
           chmod +x ./kind
           sudo mv ./kind /usr/local/bin/kind
       - name: Install GoReleaser

